src/main/java/duchess/Deadline.java

```diff
@@ -0,0 +1,48 @@
+package duchess;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task with a deadline. 
+ * A {@code Deadline} object stores the task name and the due date/time.
+ */
+public class Deadline extends Task{
```

Comment:
A space can be added here for consistency and readability.
```suggestion
public class Deadline extends Task {
```